### DISCUSSION TITLE
New mime modify tool in Rspamadm

### DISCUSSION THREAD

**Discussion by HelloooNanny** [rspamd/rspamd#3679] [Q&A]
Hi everyone,

I am new to this community and this is my first post.

I am struggling to make the mime modify command work.

Rspamd was installed as a part of a mailcow server on a VPS. Running all the latest versions.

This is what I have done:

`cd /root/mailcow-dockerized`

Then created  footer.txt and footer.html in this folder. Then:

`docker-compose exec rspamd-mailcow rspamadm mime modify -t footer.txt -H footer.html`

ButI have the following error:

```
Usage: rspamadm mime modify [-h] [-a <header=value>] [-r <header>]
[-R <header=pattern>] [-t <file>] [-H <file>]
<file> [<file>] …

Error: missing argument ‘file’
ERROR: 1

```

It would be great if someone could help me.

Thanks

**Comment by hsarcon** [comment]
Hi HelloooNanny,

this rspamdadm mime command is not for configure a header/footer in the rspam Framework, it can only directly modify an email message ([link](https://rspamd.com/doc/rspamadm.html#rspamadm-mime))

I can imagine it may be work like a alterMIME Setup, see https://baturorkun.medium.com/disclaimer-to-outgoing-emails-2ce576a6433b . 

I didnt test it yet

**Reply by vstakhov** [reply]
You can do the same with Rspamd: https://gist.github.com/vstakhov/3dda60a5638a6aefd454973bd687fc21

**Reply by oscarmlage** [reply]
@vstakhov may I ask if this `footer.lua` is still working? I was trying and the best result I got was something like this:

```
[...]
</body></html>
<br><p>html footer</p>
--=_81d5e11ca241d02414ad620aec73e4bb--
```

But Thunderbird and some other clients were not able to render it, dunno if the issue is that it's being added after `</html>` or something else (tia).
